File: JAVA PROGRAMMING/Test23.java
// 23. PRINT SUM OF ALL NUMBERS BETWEEN 0 TO N
import java.util.Scanner;
public class Test23 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        int sum = 0;
        for(int i = 0; i <= range; i++) {
            sum += i;
        }
        System.out.println("Sum Of 0 To "+range+" Is: "+sum);
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test23.ts
// 23. PRINT SUM OF ALL NUMBERS BETWEEN 0 TO N
const func23 = (range: number): void => {
    let sum: number = 0;
    for(let i = 0; i <= range; i++) {
        sum += i;
    }
    console.log(`Sum = ${sum}`);
} 
func23(10);
func23(50);
func23(100);